examples/private_set_intersection/README.md

```diff
@@ -3,10 +3,10 @@
 ## To update this example
 
 1. Build the example, including the Wasm module
-1. Push the module to GS via
+2. Push the module to GS via
    `./scripts/push-example -e private_set_intersection`
-1. Fix the URL and hash in [`./oak_app_manifest.toml`](./oak_app_manifest.toml)
-1. Sign the module with the test key that is checked in the repository (only for
+3. Fix the URL and hash in [`./oak_app_manifest.toml`](./oak_app_manifest.toml)
+4. Sign the module with the test key that is checked in the repository (only for
    test / debug use):
```

Comment:
The const numbering is intentional, the sequential numbers are assigned by the markdown renderer.